components/navbar.tsx

```diff
@@ -66,36 +64,38 @@ export default function NavBar() {
               >
                 <SearchIcon />
               </button>
-              <Link href="/about">
-                <a className="hover:opacity-60 md:px-7 px-2">
+              <nav>
+                <Link href="/about">
+                  <a className="hover:opacity-60 md:px-7 px-2">
+                    <span className="text-l font-normal mt-4 text-back dark:text-white">
+                      About
+                    </span>
+                  </a>
+                </Link>
+                <a
+                  href="https://github.com/warpdotdev/workflows#contributing"
+                  target="_blank"
+                  rel="noreferrer"
+                  className="hover:opacity-60 md:pr-7 pr-2"
+                  onClick={(_) => {
+                    gtag.event({
+                      action: "click_on_navbar_github",
+                      category: "Click on GitHub",
+                      label: "Click on GitHub via NavBar",
+                      value: window.location.pathname,
+                    });
+                  }}
+                >
                   <span className="text-l font-normal mt-4 text-back dark:text-white">
-                    About
+                    Contribute
```

Comment:
I think per Alex's feedback this should be "Contribute on GitHub"--see item 3) in his doc

Reply:
I mentioned this in the PR description, but "Contribute on github" does not fit on mobile, we might have to come up with some other content here